components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -33,7 +32,7 @@ open class GleanInternalAPI {
     // Include our singletons of StorageEngineManager and PingMaker
     private lateinit var storageEngineManager: StorageEngineManager
     private lateinit var pingMaker: PingMaker
-    private lateinit var configuration: Configuration
+    internal lateinit var configuration: Configuration
```

Comment:
Instead of changing this, can't we pass the default configuration to the uploader in the test that needs it? We're always using the default configuration in the tests anyway

Reply:
This change is needed so the EventsStorageEngine has access to `MAX_EVENTS`.  We could provide an API in Glean to get it or pass the Configuration object along to all storage engines instead...

Reply:
Ah, good point, never mind my comments about this then!